clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -385,6 +391,10 @@ public class ProducerConfig extends AbstractConfig {
                                         atLeast(0L),
                                         Importance.LOW,
                                         CommonClientConfigs.RETRY_BACKOFF_MAX_MS_DOC)
+                                .define(ENABLE_METRICS_PUSH_CONFIG,
+                                        Type.BOOLEAN, true,
```

Comment:
Formatting. Please place the type and the default on separate lines like the surrounding text.

Reply:
Missed it, thanks again.